platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/eventhandling/ConsensusRoundHandler.java

```diff
@@ -519,18 +473,10 @@ private void createSignedState() throws InterruptedException {
         ssTimingStat.stopCycle();
     }
 
-    public void addMinGenInfo(final long round, final long minGeneration) {
```

Comment:
It's so nice to remove this class's dependency on this data!

Reply:
oh yeah, makes things much simpler